app/controllers/instruments_controller.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 class InstrumentsController < ApplicationController
-  skip_before_filter :configure_api
+  skip_before_action :configure_api
 
   def search
     instrument = Instrument.find_by_barcode(params[:instrument_barcode])
-    render text: instrument.nil? ? '' : instrument.name
+    render plain: instrument.nil? ? '' : instrument.name
   end
```

Comment:
Deprecated in rails 5. text never set the correct content type. Followed recommended change to plain.